.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         run: |
           wget https://crystal-lang.org/install.sh
           chmod +x install.sh
-          sudo ./install.sh --crystal=1.0.0
+          sudo ./install.sh --version=1.1
```

Comment:
Or should we start using the container here too?

Reply:
I think we are not using the container here because it is easier (and more performant?) to install RabbitMQ in Ubuntu

Reply:
...with that said, if we want to use container for Crystal, we could switch to a service container for RabbitMQ, but it is a bit more Yaml needed then, example: https://github.com/dentarg/actions-test/blob/a721484c6b99d271d20779159bff8b3c2720fbf9/.github/workflows/rabbitmq.yml#L14-L27